app/lib/three_scale/json_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  class JSONValidator < ::JSON::Validator
+    def self.setup!
+      self.schema_reader = JSON::Schema::Reader.new(accept_uri: false, accept_file: false)
+    end
+
+    def self.setup
+      setup!
+    rescue StandardError => error
```

Comment:
I do not really see how `::setup!` could fail ...

Reply:
Then we shouldn't have it at all.

Reply:
yes well this setup stuff should really go to an initializer, configuring what you need before use